### PR TITLE
[DOC] sale_layout_hidden_section description

### PR DESCRIPTION
**Pull request by Englishrain**
[DOC] sale_layout_hidden_section description

### CODE REVIEW

File: sale_layout_hidden_section/static/description/index.html [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+</section>
+
+<section class="oe_container oe_dark">
+  <div class="oe_row oe_spaced">
+	<h4 class="oe_slogan">Usage</h4>
+    <div class="oe_span12">
+      <p class="oe_mt32">
+		<ul>
+                  <li>Open menu 'Sales' &rarr; 'Settings'</li>
+		  <li>Switch 'Sales Reports Layou't to 'Personalize the sales orders and invoice report with categories, subtotals and page-breaks'</li>
```
Comment (ilmir-k) [inline sale_layout_hidden_section/static/description/index.html]: Typo:  _Layou't_

File: sale_layout_hidden_section/static/description/index.html [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+                            color: #75526b;
+                            border: 3px dashed #75526b;
+                            float: left;
+                            padding: 4px 12px;
+                            -webkit-transform: rotate(2deg);
+                            -o-transform: rotate(2deg);
+                            -moz-transform: rotate(2deg);
+                            -ms-transform: rotate(2deg);">
+                    Tested on Odoo<br/>10.0 enterprise
+                </div>-->
```
Comment (ilmir-k) [inline sale_layout_hidden_section/static/description/index.html]: It works for enterprise too. Uncomment it.

**Comment by ilmir-k** [comment]
Set main image

**Comment by yelizariev** [comment]
Why it's still not merged?

**Comment by ilmir-k** [comment]
I'm waiting for updates from Lily. She should send new main image here

**Comment by yelizariev** [comment]
default

**Pull request merged**